packages/passport/src/index.ts

```diff
@@ -0,0 +1,296 @@
+import { createEIP1193Provider, type WalletInit } from '@web3-onboard/common'
+import { Network, Passport } from '@0xpass/passport'
+import { createPassportClient } from '@0xpass/passport-viem'
+import { http } from 'viem'
+import { mainnet } from 'viem/chains'
+import { AuthenticatedHeaders } from '@0xpass/passport/dist/types'
+import CryptoJS from 'crypto-js'
+import passportLogo from './passportlogo.svg'
+import { PassportOptions } from 'types'
```

Comment:
We will want any non devDependency to be imported dynamically so that imports are light on dapps on start and deps are only pulled in once passport is selected from the web3-onboard options.

Any types should also be imported as `import type {....} from '....'`